playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -100,7 +100,7 @@
       {{ socok8s_ext_vip }} compute.openstack.svc.cluster.local
       {{ socok8s_ext_vip }} network.openstack.svc.cluster.local
       {{ socok8s_ext_vip }} dashboard.openstack.svc.cluster.local
-      {{ socok8s_ext_vip }} novncproxy.openstack.svc.cluster.local
+      {{ socok8s_ext_vip }} nova-novncproxy.openstack.svc.cluster.local
```

Comment:
Can we make similar change in OSH side ?

https://github.com/SUSE-Cloud/socok8s/blob/master/playbooks/roles/deploy-osh/tasks/main.yml#L92

Reply:
done